string command
---
bool done
string[] reachable_urgent
string[] reachable_corridors
